packages/notification-center-vue/package.json

```diff
@@ -22,7 +22,7 @@
     "@emotion/css": "^11.10.5",
     "@novu/notification-center": "^0.10.1",
     "floating-vue": "https://github.com/LetItRock/floating-vue/raw/main/packages/floating-vue/floating-vue-2.0.0-beta.21.tgz",
-    "vue": "^3.2.45"
+    "vue-demi": "latest"
```

Comment:
here we should set a specific version because the latest might break something for us in the future if the `vue-demi` will be updated

Reply:
no problem